modules/internal/ChapelArray.chpl

```diff
@@ -2205,6 +2171,26 @@ module ChapelArray {
     var _instance; // generic, but an instance of a subclass of BaseArr
     var _unowned:bool;
 
+    proc chpl__rvfMe() param {
+      return _instance.chpl__rvfMe();
+    }
+
+    proc chpl__serialize() where chpl__rvfMe() {
+      return _instance.chpl__serialize();
+    }
+
+    proc type chpl__typeOfInstance() type {
+      var x: this;
```

Comment:
Can you not use the "static field type" primitive here as well?

Reply:
I was wondering about unifying these when setting up this PR as well, but figured I'd get it reviewed first in case I had to do other edits.  Do you prefer using the primitive vs. the helper routine?  (tradeoff seems to be "relies on a primitive" vs. "not"?)

Reply:
Mild preference for the primitive, but I wouldn't hold up the PR over it. I was more curious whether there was a meaningful difference.